crates/client/mempool/src/lib.rs

```diff
@@ -187,7 +187,7 @@ impl Mempool {
                 true,
             )?;
 
-            self.metrics.accepted_transaction_counter.add(1, &[]);
+            self.metrics.accepted_transaction_counter.add(1.0, &[]);
```

Comment:
why do we need a decimal counter here? cant we stick with an integer?

Reply:
also why an upDownCounter if we never decrement? AM I missing something?

Reply:
Yeah, initially we were going store the flow of increase and decrease of transactions in the mempool, but now we deprecated that thought, removed the `up_down_counter` logic completely